instrumentation/net/http/otelhttp/transport.go

```diff
@@ -63,19 +75,45 @@ func NewTransport(base http.RoundTripper, opts ...Option) *Transport {
 
 	c := newConfig(append(defaultOpts, opts...)...)
 	t.applyConfig(c)
+	t.createMeasures()
 
 	return &t
 }
 
 func (t *Transport) applyConfig(c *config) {
 	t.tracer = c.Tracer
+	t.meter = c.Meter
 	t.propagators = c.Propagators
 	t.spanStartOptions = c.SpanStartOptions
 	t.filters = c.Filters
 	t.spanNameFormatter = c.SpanNameFormatter
 	t.clientTrace = c.ClientTrace
 }
 
+func (t *Transport) createMeasures() {
+	var err error
+	t.requestBytesCounter, err = t.meter.Int64Counter(
+		clientRequestSize,
+		metric.WithUnit("By"),
+		metric.WithDescription("Measures the size of HTTP request messages."),
```

Comment:
Should this be `Measures the size of HTTP request messages (compressed).`?

https://github.com/open-telemetry/opentelemetry-specification/blob/855d80c93ffc11ee7c96177be8082989afb020f5/semantic_conventions/metrics/http.yaml#L56C13-L56C69

Reply:
Same for response below, and for server request/response.

Reply:
I noticed that it was `(uncompressed)` before. Isn't `http` package handling the transport compression and therefore we get the uncompressed message size value?

Reply:
Ah.  Makes sense.  We aren't really correctly implementing this metric then, are we?

Reply:
This is how I see it. I was not checking if it is possible to implement it correctly.